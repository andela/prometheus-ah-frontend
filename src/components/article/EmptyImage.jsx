import React from 'react';
import placeholder from '../../images/paceholder.jpeg';

const EmptyImage = () => (
  <img src={placeholder} alt="Author" className="image-placeholder" />
);

export default EmptyImage;
