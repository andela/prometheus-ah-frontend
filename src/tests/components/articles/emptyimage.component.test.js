import React from 'react';
import { shallow } from 'enzyme';
import EmptyImage from '../../../components/article/EmptyImage';


describe('Editor component', () => {
  it('should render Image placeholder correctly', () => {
    const wrapper = shallow(<EmptyImage />);
    expect(wrapper).toMatchSnapshot();
  });
});
