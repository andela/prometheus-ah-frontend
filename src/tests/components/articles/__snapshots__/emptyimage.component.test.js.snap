// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Editor component should render Image placeholder correctly 1`] = `
<img
  alt="Author"
  className="image-placeholder"
  src={Object {}}
/>
`;
